### PR TITLE
MTV UI Overview and Settings dialog

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.5

Resolves https://issues.redhat.com/browse/MTV-556 by adding a short description of the UI and documentation of the Overview page.

Previews: 
1. https://file.emea.redhat.com/rhoch/settings/html-single/#mtv-ui_mtv [quick description of the UI -- it really is pretty intuitive]
2. https://file.emea.redhat.com/rhoch/settings/html-single/#mtv-overview-page_mtv [The Overview page]
3. https://file.emea.redhat.com/rhoch/settings/html-single/#mtv-settings_mtv [Configuring Settings in the Overview page]

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas @bennyz Please review.

**Comment by yaacov** [comment]
LGTM

File: documentation/modules/mtv-settings.adoc
Comment (ahadas) [inline documentation/modules/mtv-settings.adoc:12]: this list aligns with the order the items appear in the UI which is good
but @yaacov, we should place the settings that are related to warm migration at the bottom - we consider dropping warm migration at some point...

File: documentation/modules/mtv-settings.adoc
Comment (yaacov) [reply documentation/modules/mtv-settings.adoc:33]: added an issue:
https://github.com/kubev2v/forklift-console-plugin/issues/705

File: documentation/modules/mtv-settings.adoc
Comment (yaacov) [reply documentation/modules/mtv-settings.adoc:12]: added an issue:
https://github.com/kubev2v/forklift-console-plugin/issues/706

this issues, 705 and 706, are easy to solve, we can update the docs now, or wait until the fixes are merged ...

Comment (ahadas) [reply documentation/modules/mtv-settings.adoc:12]: yeah, I'd wait for 706 but we can be explicit about the controller's main container before the fix for 705 gets in

**Comment by RichardHoch** [comment]
@ahadas @yaacov Please review again.

File: documentation/modules/mtv-settings.adoc
Comment (ahadas) [reply documentation/modules/mtv-settings.adoc:12]: I see the order was updated, resolving

**Review by ahadas — APPROVED** [review]

**Review by qiyuann — COMMENTED** [review]
The picture of overview page is not up to date, the fields in settings part are changed in the latest build.

**Review by yaacov — APPROVED** [review]
LGTM

a. the bit about how we check if a user is admin or not, don't know if it's important, but me as user will want to know how to give "admin" privileges to a user in this scenario.

b. the image needs update as @qiyuann commented above

**Comment by RichardHoch** [comment]
@yaacov I must have accidently deleted the information about what you can see as an admin vs. non-admin. Will add.

**Review by qiyuann — APPROVED** [review]

File: documentation/modules/mtv-overview-page.adoc
Comment (yaacov) [reply documentation/modules/mtv-overview-page.adoc:11]: handeled in another PR

**Review by yaacov — APPROVED** [review]

**Review by ahadas — APPROVED** [review]

File: documentation/modules/mtv-ui.adoc
Comment (anarnold97) [inline documentation/modules/mtv-ui.adoc:19]:
```suggestion
* If you are a non-administrator, you can see only the projects that you have permissions to work with.
```

File: documentation/modules/mtv-overview-page.adoc
Comment (anarnold97) [inline documentation/modules/mtv-overview-page.adoc:36]:
```suggestion
** Running: Whether the Operator is currently running and waiting for the next reconciliation.
```

File: documentation/modules/mtv-overview-page.adoc
Comment (anarnold97) [inline documentation/modules/mtv-overview-page.adoc:31]:
```suggestion
* Operator: The namespace on which the {project-short} Operator is deployed, and the status of the Operator.
```

Pedantic and possibly wrong

File: documentation/modules/mtv-settings.adoc
Comment (anarnold97) [inline documentation/modules/mtv-settings.adoc:33]:
```suggestion
|The interval at which a new snapshot is requested before initiating a warm migration
```
Pedantic

File: documentation/modules/mtv-settings.adoc
Comment (anarnold97) [inline documentation/modules/mtv-settings.adoc:45]: Do you need this much which space... I do not mind but thought I should highlight

**Review by anarnold97 — APPROVED** [review]
Pedantic quibbles - nothing that I would say is an issue, apart from a small typo.

**Comment by anarnold97** [comment]
Nice piece of writing, a small typo and some very pedantic quibbles that you can ignore, if you chose to do so

File: documentation/modules/mtv-overview-page.adoc
Comment (RichardHoch) [reply documentation/modules/mtv-overview-page.adoc:31]: I shall respectfully demur on this one.

File: documentation/modules/mtv-settings.adoc
Comment (RichardHoch) [reply documentation/modules/mtv-settings.adoc:45]: Thanks. Extra lines result of shifting table rows' should have removed them when finished moving text. Interesting that bccutil did not object.

**Pull request merged**
